supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesParameterConversions.scala

```diff
@@ -29,14 +29,16 @@ trait PipelinesParameterConversions {
       val labels = ActionBuilder.parameterLabels(fileInput)
       fileInput.cloudPath match {
         case drsPath: DrsPath =>
-          import cromwell.backend.google.pipelines.v2alpha1.api.ActionCommands.ShellPath
 
           import collection.JavaConverters._
 
           val drsFileSystemProvider = drsPath.drsPath.getFileSystem.provider.asInstanceOf[DrsCloudNioFileSystemProvider]
 
           val drsDockerImage = config.getString("drs.localization.docker-image")
-          val drsCommand = List(fileInput.cloudPath.escape, fileInput.containerPath.escape) ++ drsPath.requesterPaysProjectIdOption.toList
+          // Note: Don't ShellPath.escape the paths as we are directly invoking the localizer and NOT launching a shell.
+          val drsCommand =
+            List(fileInput.cloudPath.pathAsString, fileInput.containerPath.pathAsString) ++
```

Comment:
Didn't you scupper the `cloudPath`? Should this be `cloudHost`?

Reply:
They're returning the same thing at the moment. Other places are likely still using `cloudPath` too. I made minimal changes in this PR. If you would like we can polish up a little bit more, but like all things DOS/DRS I'm 100% positive we'll be back in this code _real soon_.